composer.json

```diff
@@ -27,7 +27,7 @@
         }
     ],
     "require": {
-        "php": "8.1.*||8.2.*||8.3.*||8.4.*",
+        "php": "8.1.*||8.2.*||8.3.*||8.4.*||8.5.*",
```

Comment:
As per asgrim/example-pie-extension#10 we need upstream support in php/php-windows-builder#12 first for Windows.

Side note: the PR says to allow maintainers access to push, but I can't seem to push to this PR, as I have some amendments. I may need to make a new PR if I can't figure out how to do this, sorry :(